### PR TITLE
Add test for JS function "merge_selected_cells" in test_merge_cells.py

### PR DESCRIPTION
**Pull request by AndresSan6**
This PR add test requested in #4177

### CODE REVIEW

File: notebook/tests/selenium/utils.py [hunk 1]
```diff
@@ -99,6 +99,12 @@ def focus_cell(self, index=0):
         cell.click()
         self.to_command_mode()
         self.current_cell = cell
+    
+    def select_command_lines(self, initial_index=0, final_index=0):
```
Comment (takluyver) [inline notebook/tests/selenium/utils.py]: Let's call this something like `select_cell_range`.

**Comment by takluyver** [comment]
Other than the naming thing, this looks good, thanks.

**Comment by AndresSan6** [comment]
Done!

**Comment by takluyver** [comment]
@askerry the display isolation test you added in PR #4146 has failed on Travis CI with this error:

```
        # The second rectangle will be black
>       notebook.browser.switch_to.frame(iframes[2])
E       IndexError: list index out of range
```

Could you take a look and figure out what might be going wrong?

**Comment by askerry** [comment]
> @askerry the display isolation test you added in PR #4146 has failed on Travis CI with this error:
> 
> ```
>         # The second rectangle will be black
> >       notebook.browser.switch_to.frame(iframes[2])
> E       IndexError: list index out of range
> ```
> Could you take a look and figure out what might be going wrong?

huh yes, will check it out. weird, since it passed Travis CI build on my PR.

**Comment by takluyver** [comment]
Intermittent failures in UI tests usually mean there's a race condition somewhere ;-)

**Comment by takluyver** [comment]
Thanks @AndresSan6, and congratulations on your first contribution to the notebook. :-)

**Comment by AndresSan6** [comment]
Thanks for accepting my PR :+1: will be around to look which other issues I can help on!

**Pull request merged**